lib/board.ex

```diff
@@ -21,9 +21,13 @@ defmodule Board do
   end
 
   def get_tile(board, {row_index, col_index}) do
-    board
-    |> Enum.at(row_index)
-    |> Enum.at(col_index)
+    if out_of_bounds?(board, {row_index, col_index}) do
+      nil
+    else
```

Comment:
I'm more familiar with Java & Scala but seeing a function return `nil` gives me a bit of pause. Are callers of this function handling the `nil` case?

Reply:
@klkelley thank you! Only one caller of this function is handling the `nil` case, so it makes a lot of sense that this is iffy. I've updated it [here](https://github.com/lottetreg/minesweeper/commit/cdd7f5f798bc79fbf1e4c9c807fc313b78040d41). I should probably change `Board.get_tile/2` to account for cases in which the tile doesn't exist, and then update the callers to handle those cases, right?

Reply:
Nice, yeah I'd still account for the case the tile doesn't exist